### ISSUE TITLE
Discussion on training

### ISSUE THREAD

**Issue by Du-Yao** [vvvm23/sundae#4]
I have reproducede the paper based your code, and I tried train it on dataset Text8. The loss and accuracy was about 2.6 and 28 respectively, and it hard to improve. So I would like to ask you how many steps did you train and what about the metrics? Thank you very much!

**Comment by vvvm23** [comment]
Hi,
The default config should do better than that. Can you check if your python environment matches the `environment.yaml` in this repo? There was a similar issue before that had a mismatch in the `x-transformers` version.

**Comment by Du-Yao** [comment]
Thanks for your reply. I confirm my conda environment and try it again, then it works. In addition, I found the model could converge quickly when enhanced the learning rate to 1e-4. Thanks again.

**Comment by vvvm23** [comment]
Good to hear it works ~ I need to figure out why the version change breaks training, but haven't had the time (*´ω｀*)
